commitlint.config.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* This is the pattern from semantic-release */
+const automaticCommitPattern = /^chore\(release\):.*\[skip ci\]/;
+
+/** @type {import('@commitlint/types').UserConfig} */
+module.exports = {
```

Comment:
Nit: `commitlint` [supports](https://commitlint.js.org/reference/configuration.html) `.mjs` and `.ts` config files. Can we move this to an ESM format?

Reply:
All suggested changes are adopted. Thanks for detailed review!